docs/en/searching-blocks.md

```diff
@@ -0,0 +1,32 @@
+# Searching Blocks
+
+## Overview
+
+Content created through SilverStripe can be searched in various ways.
+A popular option is the [silverstripe/fulltextsearch](https://github.com/silverstripe/silverstripe-fulltextsearch)
+module, which integrates with search services such as Solr.
+
+## Usage
+
+When you install the fulltextsearch module, it will auto-discover
+an index provided by elemental: 
+`DNADesign\Elemental\Search\ElementalSolrIndex`.
+
+It works based on a new `getElementsForSearch` method added to your `Page` class
+through the `DNADesign\Elemental\Extensions\ElementalPageExtension`.
+This method renders out the full content of all elements,
+strips out the HTML, and indexes it as one field.
+
+## Configuration
+
+In many cases, you'll already have configured your own search index.
+Since the `ElementalSolrIndex` is auto-discovered, it duplicates
+search indexing effort even when it is unused.
+
+You can disable it via YAML config in favour of your own index definition:
+
+```yml
+SilverStripe\FullTextSearch\Search\FullTextSearch:
+  indexes:
+    - MyCustomIndex
```

Comment:
Wouldn't this just merge MyCustomIndex into whatever was already configured?

Reply:
It would if there's other explicit definitions of this config array, but usually there aren't. Note that this advice is modeled off the existing docs in https://github.com/silverstripe/cwp/blob/master/docs/en/02_Features/01_Solr_search/03_Basic_usage.md#using-multiple-indexes. See https://github.com/silverstripe/silverstripe-fulltextsearch/blob/3/src/Search/FullTextSearch.php#L46.